community/contributors/contributors-name-mapping.json

```diff
@@ -54,6 +55,7 @@
   "kariem": "Kariem Hussein",
   "kevinpollet": "Kevin Pollet",
   "khalidq": "Khalid",
+  "koentsje": "Koen Aers",
   "lalmeras": "Laurent",
   "lordofthejars": "Alex Soto",
   "lukasz-antoniak": "Lukasz Antoniak",
```

Comment:
can we add 
```
"lucamolteni":  "Luca Molteni", 
```

as well?

Reply:
Sure :) In fact I'll submit another PR with all these unrelated changes.

Reply:
#232

Reply:
Added to #232